### ISSUE TITLE
Lack of color contrast for the project version and license selected on the Licenses page

### ISSUE THREAD

**Issue by joaopalmeiro** [jupyterlab/jupyterlab#16580]
Hi! 👋 

## Description

In JupyterLab 4.3.0a2, there is information with a lack of color contrast in the table on the Licenses page:

<img width="1582" alt="Screenshot 2024-07-12 at 15 33 34" src="https://github.com/user-attachments/assets/0f5ce71d-6d22-4e77-8677-f7cc50384b97">

## Reproduce

1. Go to _Help_ > _Licenses_ in the top menu.
2. Select a package from the license table.

## Expected behavior

The project version and license text color contrasts with the background. For example:

<img width="1582" alt="Screenshot 2024-07-12 at 15 47 09" src="https://github.com/user-attachments/assets/795091e8-7a26-495c-a715-f3e8463fe8e7">

or

<img width="1582" alt="Screenshot 2024-07-12 at 15 47 34" src="https://github.com/user-attachments/assets/bc21949d-7706-4c7e-b094-efbb1921b258">

## Context

- Operating System and version: macOS Sonoma 14.5
- Browser and version: Chrome 126.0.6478.127
- JupyterLab version: 4.3.0a2
- Python version: 3.10.13

Let me know what you think and if I can open a PR. Thanks!

**Comment by krassowski** [comment]
Sure, with things like this please welcome to fire a PR immediately after opening an issue :)

> Note to self:  remember to test it in the dark mode when reviewing the PR

**Comment by joaopalmeiro** [comment]
Thanks! Sure!
